packages/ibm-products/src/components/Guidebanner/GuidebannerElementButton.tsx

```diff
@@ -18,23 +18,58 @@ import { pkg } from '../../settings';
 const blockClass = `${pkg.prefix}--guidebanner__element-button`;
 const componentName = 'GuidebannerElementButton';
 
+interface GuidebannerElementButtonProps {
+  /**
+   * Provide the contents of the GuidebannerElementLink.
+   */
+  children: ReactNode;
+
+  /**
+   * Provide an optional class to be applied to the containing node.
+   */
+  className?: string;
+
+  /**
+   * If type is "primary", then return a tertiary button with the "crossroads" icon,
+   * else return a ghost button.
+   */
+  type?: string;
+}
+
 /**
  * One of two buttons styled specifically for the GuidebannerElement.
  */
-export let GuidebannerElementButton = ({
-  children,
-  className,
-  type,
-  ...rest
-}) => {
-  if (type === 'primary') {
+export let GuidebannerElementButton = React.forwardRef<
+  Button,
+  GuidebannerElementButtonProps
+>(
+  (
+    { children, className, type, ...rest }: GuidebannerElementButtonProps,
+    ref
+  ) => {
+    if (type === 'primary') {
+      return (
+        <Button
+          {...rest}
+          className={cx(blockClass, className)}
+          iconDescription={'Crossroads'}
```

Comment:
Can you update this so it's not a hard coded string? This component might need a new prop, `iconDescription`